src/utils/strings.ts

```diff
@@ -26,3 +26,18 @@ const english_ordinal_rules = new Intl.PluralRules('en', { type: 'ordinal' });
 export function ordinal(value: number) {
   return suffixes[english_ordinal_rules.select(value)];
 }
+
+// Only handles cases where the plural form has a suffix appended. Does not handle cases like baby => babies, person => people
+export const maybePluralizeOnlyNoun = (
```

Comment:
we should really start thinking about shared repo, these are great to be the first shared functions

Reply:
yepp... totally agree, maybe a Cortex npm utils package